src/Abp/Localization/CultureInfoHelper.cs

```diff
@@ -33,23 +33,6 @@ public static IDisposable Use([NotNull] CultureInfo culture, CultureInfo uiCultu
             });
         }
 
-        /// <summary>
-        /// This method is a temporary solution since CultureInfo.GetCultureInfo() does not exists in netstandard yet.
-        /// </summary>
-        [Obsolete("Use CultureInfo.GetCultureInfo instead!")]
-        public static CultureInfo Get(string name)
-        {
-            if (name == null)
-            {
-                return null;
-            }
-
-            return Cache.GetOrAdd(name, n =>
-            {
-                return new CultureInfoCacheEntry(CultureInfo.ReadOnly(new CultureInfo(n)));
-            }).CultureInfo;
-        }
-
         private class CultureInfoCacheEntry
```

Comment:
It appears that this private class can be removed too.

Reply:
Nice catch, cleaned that up as well.